### PR TITLE
Apply user tabs improvements

### PR DESCRIPTION
**Pull request by AnnieLeonia**

### CODE REVIEW

File: client/src/components/UserSettings.vue [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
-        <i class="fas fa-question-circle fa-2x"></i> HELP
+        HELP
       </a>
       <a href="/auth/logout" class="log-out">
-        <i class="far fa-hand-point-right fa-2x"></i> LOG OUT
+        LOG OUT
       </a>
       <a class="user-remove" @click="deleteMe">
-        <i class="fas fa-trash-alt fa-2x"></i> DELETE ACCOUNT
+        DELETE ACCOUNT
```
Comment (VictorWinberg) [inline client/src/components/UserSettings.vue:12]: Nice, removal of icons! Not necessary with icons.

File: client/src/components/Header.vue [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <header>
+  <header id="header">
```
Comment (VictorWinberg) [inline client/src/components/Header.vue:2]:
```suggestion
  <header>
```
If `id="header"` is not used anymore, we can remove it

File: client/src/views/User.vue [hunk 3]
```diff
@@ -1,82 +1,73 @@
...
-          {{ user.lvl != null ? user.lvl : "?" }}
-        </span>
-      </i>
-      <div class="user-xp__bar">
-        <span class="user-xp__text" :style="xpTextStyle(user.lvl)">
-          {{ user.lvlXp != null ? user.lvlXp : "-" }} /
-          {{ user.reqLvlXp != null ? user.reqLvlXp : "-" }} xp
-        </span>
+  <div ref="userWrapper" class="user-wrapper">
+    <div ref="userProfile">
```
Comment (VictorWinberg) [inline client/src/views/User.vue:3]:
```suggestion
    <div>
```
If `ref="userProfile"` is not used anymore, we can remove it

File: client/src/views/User.vue [hunk 4]
```diff
@@ -100,19 +91,31 @@ export default Vue.extend({
...
+        { id: 3, icon: "fa fa-cog", component: "user-settings" }
+      ]
     };
   },
   computed: {
     ...mapState("user", ["user"])
   },
+  mounted() {
+    this.tabContentHeight =
+      this.$refs.userWrapper?.clientHeight - this.$refs.tabs?.clientHeight;
```
Comment (VictorWinberg) [inline client/src/views/User.vue:109]: Curious, what is the reason for using `$refs` with `ref` instead of `querySelector` with `.classname`?

**Review by VictorWinberg — APPROVED** [review]
LGTM :rocket:

File: client/src/views/User.vue (hunk 4 above)
Comment (AnnieLeonia) [reply client/src/views/User.vue:109]: https://forum.vuejs.org/t/difference-between-refs-and-queryselector/35504/2

**Pull request merged**